main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    for genre in ("Western", "Action", "Dramma"):
+        Genre.objects.create(name=genre)
+
+    for first_name, last_name in (
+            ("George", "Klooney"),
+            ("Kianu", "Reaves"),
+            ("Scarlett", "Keegan"),
+            ("Will", "Smith"),
+            ("Jaden", "Smith"),
+            ("Scarlett", "Johansson")
+    ):
```

Comment:
It's better to create separate variables to store genres list and actors tuples.